### DISCUSSION TITLE
2 manual consul playing 3+ manual dispositions

### DISCUSSION THREAD

**Discussion by Penrhos1920** [GrandOrgue/grandorgue#1507] [Help]
I only have 2 manuals on my system, but would like to be able to play 3+ manuals by using pistons to select which manual each couple to.  I used to be able to do this with the free version of Hauptwerk, but it took a lot of concentration to set it up for each organ.  Could someone please point me in the right direction.  Thank you

**Comment by larspalo** [comment]
Have a look at https://github.com/GrandOrgue/grandorgue/discussions/1364 to find a few different suggestions as there are many ways to do it.

**Reply by Penrhos1920** [reply]
Thanks.  Got it working, but with a small issue that M1-M2 and M2-M3 couplers don't work.  M2-M3 does work!  

Will try using coupling manuals tomorrow.
